vaadin-testbench-unit-quarkus/src/test/java/com/vaadin/testbench/unit/quarkus/SecurityTestConfig.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2000-2024 Vaadin Ltd
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ * See <https://vaadin.com/commercial-license-and-service-terms> for the full
+ * license.
+ */
+package com.vaadin.testbench.unit.quarkus;
+
+import jakarta.enterprise.event.Observes;
+
+import com.testapp.security.AnyRoleView;
+import com.testapp.security.LoginView;
+import com.testapp.security.ProtectedView;
+import com.testapp.security.RoleRestrictedView;
+import io.quarkus.arc.profile.IfBuildProfile;
+import io.quarkus.test.junit.QuarkusTestProfile;
+
+import com.vaadin.flow.router.RouteConfiguration;
+import com.vaadin.flow.server.ServiceInitEvent;
+import com.vaadin.flow.server.auth.NavigationAccessControl;
+import com.vaadin.flow.server.auth.ViewAccessChecker;
+import com.vaadin.testbench.unit.quarkus.mocks.MockQuarkusServletService;
+
+// Empty configuration class used only to be able to bootstrap spring
```

Comment:
nit: should mention Quarkus probably

Reply:
Can be completely removed. It's a copy/paste from Spring add-on test

Reply:
removed